package-lock.json

```diff
@@ -1,15 +1,16 @@
 {
   "name": "react-imgix",
-  "version": "9.4.0",
+  "version": "9.5.0",
```

Comment:
Thought it was worth calling out that the lock file was out of sync before this.